addons/core/XEH_postInit.sqf

```diff
@@ -2,75 +2,27 @@
 	// Suspend the whole damn thing until the required CBA Settings are available
 	waitUntil {
 		sleep 0.5;
-		!(isNil "cba_settings_ready")
+		!(isNil "cba_settings_ready");
```

Comment:
According to Arma guidelines, semi colons should be omitted when followed by the closing of a script block closure thing. So, that's why it wasn't there. Was it causing issues?

Reply:
I had added another check waiting for an asset to load during debugging, and had neglected to remove the semi colon after I had finished, sorry!